packages/abi-typegen/package.json

```diff
@@ -43,12 +43,11 @@
     "dist"
   ],
   "scripts": {
-    "pretest": "run-s build:forc-callpaths build:forc",
+    "pretest": "run-s build build:forc",
```

Comment:
How come we are building in pretest now? Seems to me that this is probably the reason for the [flakiness](https://github.com/FuelLabs/fuels-ts/issues/2967#issuecomment-2298515385). If pretest executes concurrently packages dependent on typegen _could_ fail if the build has not finished in time.

Reply:
Thanks a lot! 🙏 
- https://github.com/FuelLabs/fuels-ts/pull/2990